### DISCUSSION TITLE
Add option to process score statistics from replay in background

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#26580] [Ideas] [answered]
Currently, there are some score statistics that requires you to play a replay to then process it. It can get really tiring as you need to go through the replay player each time for every score. I would suggest to add a button to process the replay in background below the "More statistics available after watching a replay" message. After the processing, the missing statistics would then appear on the screen. I don't know if it's possible or if it's too complex, as it would probably need to implement a new replay player but without the display part. But if it's possible, I think it would be really cool.

**Comment by peppy** [comment]
That's the eventual plan yes.

**Comment by Theighlin** [comment, marked as answer]
#14472

**Reply by felipemarins** [reply]
Thank you for the link. I didn't see it when I was searching for discussions about this. Maybe I was using the wrong keywords.
